UnitsNet.Tests/CustomCode/MassFlowTests.cs

```diff
@@ -60,6 +60,8 @@ public class MassFlowTests : MassFlowTestsBase
 
         protected override double MegapoundsPerMinuteInOneGramPerSecond => 0.132277e-6;
 
+        protected override double KilogramsPerMinuteInOneGramPerSecond => 0.01666667e-3;
```

Comment:
[This source](http://www.kylesconverter.com/mass-flow/grams-per-second-to-kilograms-per-minute) says `0.060000000000000005`?

Reply:
Good catch. I accidentally did the conversion the other way... Sorry!